int32 memoire
---
bool ST8
bool ST9
bool ST10
bool ST11
bool ST12
bool ST13
bool ST14
bool ST15
bool ST16
bool ST17
bool R5D
bool R6D
bool R7D
bool R8D
bool R5G
bool R6G
bool R7G
bool R8G
bool PI3
bool PI4
bool PI5
bool PI6
bool D5
bool D6
bool D7
bool D8
bool V5
bool V6
bool V7
bool V8
bool PS8
bool PS9
bool PS10
bool PS11
bool PS12
bool PS13
bool PS14
bool PS15
bool PS16
bool PS17
bool D5D
bool D6D
bool D7D
bool D8D
bool D5G
bool D6G
bool D7G
bool D8G
bool CPI3
bool CPI4
bool CPI5
bool CPI6
bool CP4
bool CP5
bool CP6
bool CP7
